components/mediators/datamapper/org.wso2.carbon.mediator.datamapper.engine/src/main/java/org/wso2/carbon/mediator/datamapper/engine/core/executors/ScriptExecutor.java

```diff
@@ -86,4 +90,8 @@ private void injectInputVariableToEngine(String inputSchemaName, String inputVar
         scriptEngine.eval("var input" + inputSchemaName.replace(':', '_').replace('=', '_').replace(',', '_') + "="
                 + inputVariable);
     }
+
+    private void injectPropertiesToEngine(String properties) throws ScriptException {
+        scriptEngine.eval("var " + PROPERTIES_OBJECT_NAME + "= " + properties);
```

Comment:
It is often good to change '=' things to constants. and this line has formatting issue